geoportailv3/static/js/olcs/zoomtoextent.js

```diff
@@ -0,0 +1,33 @@
+goog.provide('app.olcs.ZoomToExtent');
+
+goog.require('ol.control.ZoomToExtent');
+
+
+app.olcs.ZoomToExtent = class extends ol.control.ZoomToExtent {
+  /**
+   * @param {ol.Extent} extent Extent
+   * @param {app.olcs.Lux3DManager} ol3dm Manager getter
+   */
+  constructor(extent, ol3dm) {
+    super({label: '\ue01b', extent});
+    /**
+     * @const {app.olcs.Lux3DManager}
+     * @private
+     */
+    this.ol3dm = ol3dm;
+  }
+
+  /**
+   * @override
+   */
+  handleZoomToExtent() {
+    if (this.ol3dm && this.ol3dm.is3dEnabled()) {
+      const cameraExtentInRadians = [5.31, 49.38, 6.64, 50.21].map(ol.math.toRadians);
```

Comment:
My comment asked to get `cameraExtentInRadians` from `ol3dm` config.
`this.ol3dm.cameraExtentInRadians`
Here it's duplicated.

Reply:
OK, see https://github.com/Geoportail-Luxembourg/geoportailv3/pull/1933.